src/widgets/attachments/attachments-overview.component.tsx

```diff
@@ -65,10 +67,52 @@ export default function AttachmentsOverview() {
     }
   }
 
+  function handleDragOver(e: React.SyntheticEvent) {
+    e.preventDefault();
+    e.stopPropagation();
+  }
+
   function handleCurrentImageChange(index) {
     setCurrentImage(index);
   }
 
+  function handleImageSelect(index, image) {
```

Comment:
Is `image` used below? `index` is `number`, right?

Reply:
Image is not used but that's the function signature required by the library.

Reply:
Okay, but then you can leave out `image`. Unused trailing arguments can always be removed.